README.md

```diff
@@ -1 +1,156 @@
-# AwesomeBooks
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
```

Comment:
- [ ] Kindly remove the Microvere logo. If you have your logo, you can use it, otherwise, kindly remove the logo. 👍